packages/cxl-jwplayer/src/components/cxl-jwplayer.js

```diff
@@ -1,17 +1,152 @@
-import { LitElement, html, customElement } from 'lit-element';
+import { LitElement, html, customElement, property } from 'lit-element';
+import '@conversionxl/cxl-lumo-styles';
+// import { registerGlobalStyles } from '@conversionxl/cxl-lumo-styles/src/utils';
 import cxlJwplayerStyles from '../styles/cxl-jwplayer-css.js';
+// import cxlJwplayerGlobalStyles from '../styles/global/cxl-jwplayer-css.js';
 
 @customElement('cxl-jwplayer')
 export class CxlJwPlayer extends LitElement {
   static get styles() {
     return [cxlJwplayerStyles];
   }
 
-  static get properties() {
-    return {};
+  @property({ type: Object })
+  player = null;
+
+  @property({ type: Object })
+  config = {};
+
+  @property({ type: String })
+  mediaId = '';
+
+  constructor() {
+    super();
+    this.jwplayer = window.jwplayer || null;
+    if (!this.jwplayer) console.error('JWPlayer library not found.');
+  }
+
+  /*
+   * Implemented to avoid default, which attaches a shadowDOM tree byt default.
+   */
+  createRenderRoot() {
+    return this;
   }
 
   render() {
-    return html``;
+    return html`
+      <h2>cxl-jwplayer</h2>
+      <div id="cxl-jwplayer-container" class="cxl-jwplayer-container">
+        <div id="cxl-jwplayer" class="cxl-jwplayer jwplayer"></div>
+        ${this.config.transcript
+          ? html`
+              <div id="cxl-transcript-container" class="transcript-container">
+                <div id="searchbox" class="searchbox">
+                  <input
+                    id="search"
+                    type="search"
+                    class="search"
+                    autocomplete="off"
+                    results="5"
+                    placeholder="Type and press enter to search"
+                  />
+                  <span id="match" class="match hidden">0 of 0</span>
+                </div>
+                <div class="jw-aspect jw-reset"></div>
+                <div class="transcript-wrapper">
+                  <div id="transcript" class="transcript"></div>
+                </div>
+              </div>
+            `
+          : ''}
+      </div>
+    `;
+  }
+
+  firstUpdated() {
+    // Run player setup if minimal requirements are met
+    // Bail if jwplayer is not loaded or mediaId is not set
+    if (!this.jwplayer || !this.mediaId) return;
+
+    this.setupPlayer();
+  }
+
+  setupPlayer() {
```

Comment:
Not sure if it matters as this is a component and code is encapsulated, but need to ensure we can load multiple videos per page.


Latest player setup code from live site:
```
((jwplayer, data) => {
                    /**
                     * Configure and Setup the Player.
                     */
                    const media_id = REPLACE_VIA_COMPONENT_ATTR;
                    const player_setup = data[media_id]; //media object provided by php
                    const config = player_setup.media_config;
                    config.plugins = {};

                    /**
                     * Load plugins, plugins registered with JW must be camelcase and contain no spaces or dashes.
                     * We load other helper plugins that follow dash case. These aren't to be loaded directly by JW.
                     * We can change the casing, if we want to change following indexOf to 'cxl-jw' to signify plugin
                     * for JW to load.
                     */
                    player_setup.plugins.forEach( plugin => {
                        if( plugin.indexOf('-') !== -1 ) {
                            return;
                        }
                        config.plugins[plugin] = true;
                    });

                    jwplayer(`cxl_jwplayer_${media_id}`).setup(config);

                })(jwplayer, cxlJWPlayerData);
            });
```

Reply:
I'll test it carefully, especially because I don't really know if disabling shadowDOM could affect this. But, in principle, shouldn't be a problem.